modules/uid2IdSystem.js

```diff
@@ -117,8 +118,13 @@ export const uid2IdSubmodule = {
       }
     },
   },
+
 };
 
+export function attachTokenGenerator(tokenGeneratorModule) {
```

Comment:
Does a publisher who wishes to use CSTG need to call this, or is it just for tests?

Reply:
No, the publisher don't need to call this, this is for async submodule loading. no matter which module loads first, prebid core will call this function and pass the uid2Cstg in here.

https://github.com/prebid/Prebid.js/blob/master/src/hook.js#L36C8-L49